src/InlineError/InlineError.js

```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { dcnb } from "cnbuilder";
+import Icon from "react-hero-icon";
+
+export const InlineError = ({ className, text, icon, iconType, isError }) => {
+  const iconProps = { height: 20, width: 20 };
+
+  // Default Icon.
+  let defaultIcon = (
+    <Icon
+      icon="x"
+      type="sold"
+      aria-hidden="true"
+      className="su-text-digital-red su-inline su-mr-3"
+      {...iconProps}
+    />
+  );
+
+  if (icon) {
+    defaultIcon = (
+      <Icon
+        icon={icon}
+        type={iconType}
+        aria-hidden="true"
+        className="su-text-digital-red su-inline su-mr-3"
+        {...iconProps}
+      />
+    );
+  }
+
+  return (
+    <>
```

Comment:
This fragment wrapper is unnecessary since we're returning a single node conditionally. You can optionally return `null` to render nothing and keep the (virtual) DOM a little cleaner.